src/components/try-it/call-response/CallResponse.tsx

```diff
@@ -51,3 +55,42 @@ function CopyRequestUrl({ url }: CopyRequestUrlProps) {
         </button>
     );
 }
+
+type CopyResponseButtonProps = {
+    json: string;
+};
+
+function CopyResponseButton({ json }: CopyResponseButtonProps) {
+    const [ copied, setCopied ] = useState(false);
+    const { theme } = useTheme();
+
+    useEffect(() => {
+        const timeout = setTimeout(() => setCopied(false), 2000);
+        return () => clearTimeout(timeout);
+    }, [copied]);
+
+    function copyResponse() {
+        setCopied(true);
+        navigator.clipboard.writeText(JSON.stringify(json, null, 4));
+    }
+
+    return (
+        <button
+            className="absolute rounded-lg right-8 top-20 btn btn-ghost"
```

Comment:
Adding `btn-circle` and removing `rounded-lg` looks a bit better for an icon inside of a button.